app/assets/stylesheets/catarse_bootstrap/_main.scss

```diff
@@ -634,15 +634,14 @@ p {
   background-color: #3f4752;
 }
 .link-footer {
+  @include link-footer-base;
```

Comment:
This file is generated automatically by http://catarse.webflow.io/
So we'd rather have this PR without changes to the CSS